modules/media_event_dispatcher/tests/src/Unit/Media/OEmbedResourceUrlAlterEventTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Drupal\Tests\media_event_dispatcher\Unit\Media;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\media\OEmbed\Provider;
+use Drupal\hook_event_dispatcher\HookEventDispatcherInterface;
+use Drupal\media_event_dispatcher\Event\Media\OEmbedResourceUrlAlterEvent;
+use Drupal\Tests\hook_event_dispatcher\Unit\HookEventDispatcherManagerSpy;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Class OEmbedResourceUrlAlterEventTest.
+ *
+ * @package Drupal\Tests\media_event_dispatcher\Unit\Media
+ *
+ * @group hook_event_dispatcher
+ */
+class OEmbedResourceUrlAlterEventTest extends UnitTestCase {
+
+  /**
+   * The manager.
+   *
+   * @var \Drupal\Tests\hook_event_dispatcher\Unit\HookEventDispatcherManagerSpy
+   */
+  private $manager;
+
+  /**
+   * Sets up the test.
+   */
+  public function setUp() {
+    $builder = new ContainerBuilder();
+    $this->manager = new HookEventDispatcherManagerSpy();
+    $builder->set('hook_event_dispatcher.manager', $this->manager);
+    $builder->compile();
+    \Drupal::setContainer($builder);
+  }
+
+  /**
+   * Get a mock oEmbed provider.
+   *
+   * @return \Drupal\media\OEmbed\Provider
+   *   A mock Vimeo oEmbed provider.
+   */
+  protected function getMockProvider() {
+    return new Provider('Vimeo', 'https://vimeo.com/', [
+      [
+        'url'     => 'https://vimeo.com/api/oembed.json',
+        'schemes' => [
+          'https://vimeo.com/*',
+        ],
+      ],
+    ]);
+  }
+
+  /**
+   * OEmbedResourceUrlAlterEvent parsed URL alter test.
+   *
+   * This tests altering the parsed URL array for an oEmbed data request.
+   */
+  public function testOembedResourceUrlAlterEventParsedUrlAlter() {
```

Comment:
Why does this say `EventParsedUrlAlter`?

Reply:
I guess in my head I thought of it as describing both the event (```OembedResourceUrlAlterEvent``` in the method name), and then the test on that event (```ParsedUrlAlter```), but I agree that it's redundant, especially now that I consider the class name of the test as well.